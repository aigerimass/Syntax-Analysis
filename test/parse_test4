Func_with_many_args(a; b; c; d; e;){
    return a + b + c + d + e;
}

Main () {
   v = Func_with_many_args(1; 1; 1; 1;);
}
