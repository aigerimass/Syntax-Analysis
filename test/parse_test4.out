----------SYNTAX-ANALYSIS----------
Function Definition: [
name = "Func_with_many_args",
args: "a", "b", "c", "d", e
body:
>Return(Plus(Plus(Plus(Plus(Var "a", Var "b"), Var "c"), Var "d"), Var "e"));

]
--------
Function Definition: [
name = "Main",
args: 
body:
>Bind(Var "v", >Call(name: "Func_with_many_args", args: 1, 1, 1, 1));

]
--------
----------INTERVAL-ANALYSIS----------
ERROR: VARIABLE "e" doesn't exist
